src/CodeGeneration/ClassDetailsResolver/AppClassDetailsResolver.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Dealroadshow\Bundle\K8SBundle\CodeGeneration\ClassDetailsResolver;
+
+use Dealroadshow\Bundle\K8SBundle\CodeGeneration\ClassDetails;
+use Dealroadshow\Bundle\K8SBundle\Util\Str;
+use Dealroadshow\K8S\Framework\Project\ProjectInterface;
+use ReflectionObject;
+
+class AppClassDetailsResolver
```

Comment:
I think that we already have namespace `ClassDetailsResolver`, so classes may have simpler names, like AppResolver